.github/workflows/dependabot-auto-approve-and-merge.yml

```diff
@@ -0,0 +1,24 @@
+name: "Dependabot auto approve and merge"
+on:
+  pull_request_target:
+    types: [labeled]
+jobs:
+  worker:
+    runs-on: ubuntu-latest
+    if: |
+      github.actor == 'dependabot[bot]' &&
+      ( github.event.label.name == 'major' || github.event.label.name == 'minor' || github.event.label.name == 'patch' )
```

Comment:
oh boy, auto merging major changes 😟  😂

Reply:
The tests should pick it up right? RIGHT? 😱

Reply:
Yes totally 👍